test/e2e_test.sh

```diff
@@ -16,41 +16,58 @@
 
 set -ex
 
-echo "copying rekor repo"
 pushd $HOME
-if [[ ! -d rekor ]]; then
-   git clone https://github.com/sigstore/rekor.git
-else
-   pushd rekor
-   git pull
-   popd
-fi
-cd rekor
 
-echo "starting services"
-docker-compose up -d
-
-count=0
-
-echo -n "waiting up to 60 sec for system to start"
-until [ $(docker-compose ps | grep -c "(healthy)") == 3 ];
-do
-    if [ $count -eq 6 ]; then
-       echo "! timeout reached"
-       exit 1
+echo "downloading service repos"
+for repo in rekor fulcio; do
+    if [[ ! -d $repo ]]; then
+        git clone https://github.com/sigstore/${repo}.git
     else
-       echo -n "."
-       sleep 10
-       let 'count+=1'
+        pushd $repo
+        git pull
+        popd
     fi
 done
 
+echo "starting services"
+export FULCIO_METRICS_PORT=2113
+for repo in rekor fulcio; do
```

Comment:
Using a local instance of Fulcio is great - We don't have any tests currently using it, and we'll need to set up a mock oidc provider, but this will let us have much more rich e2e tests that exercise the certifciate issuance path